lib/wallet/wallet.js

```diff
@@ -4276,7 +4276,7 @@ class Wallet extends EventEmitter {
 
   async _send(options, passphrase) {
     const mtx = await this.createTX(options, true);
-    return this.sendMTX(mtx, passphrase);
+    return this.sendMTX(mtx, passphrase, true);
```

Comment:
1) We only want bids to ever be blocking so this should probably be `false`.
2) Besides that, this parameter seems unused could you do another self review and clean extra parameters like this up?

Reply:
fixed ...